### PR TITLE
[Tizen.System] Enhance API description

### PR DESCRIPTION
**Pull request by germonado**
### Description of Change ###

Add more detailed API descriptions to the modules below.
- Tizen.System.Device
- Tizen.System.Feedback

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.System.Feedback/Feedback/Feedback.cs [hunk 1]
```diff
@@ -596,7 +601,7 @@ public uint GetCountOfThemeInternal(FeedbackType type)
         }
 
         /// <summary>
-        /// Gets the id of theme selected.
+        /// Gets the current ID of the theme selected from available themes described in the conf file.
```
Comment (ppdl) [inline src/Tizen.System.Feedback/Feedback/Feedback.cs:604]:
```suggestion
        /// Gets the current id of the theme selected from available themes described in the conf file.
```

Keep consistency, id or ID?

File: src/Tizen.System/Device/Battery.cs [hunk 2]
```diff
@@ -76,11 +76,20 @@ public static class Battery
     {
         private static readonly object s_lock = new object();
         /// <summary>
-        /// Gets the battery charge percentage.
+        /// Gets the current device'sinvalid battery charge percentage as an interger value.
```
Comment (ppdl) [inline src/Tizen.System/Device/Battery.cs:79]:
```suggestion
        /// Gets the current device's invalid battery charge percentage as an interger value.
```

**Review by chanwoochoi — APPROVED** [review]

File: src/Tizen.System.Feedback/Feedback/Feedback.cs (hunk 1 above)
Comment (germonado) [reply src/Tizen.System.Feedback/Feedback/Feedback.cs]: I changed as "id" to keep consistency thanks.

File: src/Tizen.System/Device/Battery.cs (hunk 2 above)
Comment (germonado) [reply src/Tizen.System/Device/Battery.cs]: I changed that description as you suggested.

**Review by ppdl — APPROVED** [review]

**Review by chanwoochoi — APPROVED** [review]

**Pull request merged**